core/src/main/java/io/cucumber/core/cli/CommandlineOptions.java

```diff
@@ -0,0 +1,70 @@
+package io.cucumber.core.cli;
+
+import org.apiguardian.api.API;
+
+/**
+ * Contains all available command line options for {@link io.cucumber.core.cli.Main}
+ * <p>
+ * After being passed to {@link io.cucumber.core.cli.Main#main} function, these options will be
+ * parsed by {@link io.cucumber.core.options.CommandlineOptionsParser} to provide running options
+ * to CLI.
+ * <p>
+ * All the options are defined as static string variables to allow other programs to call
+ * {@link io.cucumber.core.cli.Main#main} function in a more consistent way.
+ * E.g. {@code io.cucumber.core.cli.Main.main(NAME, "TestName", THREADS, 2)}
```

Comment:
Note that Main.main will call ` System.exit(exitStatus)`. This may not be desirable for most users. Would it be worth adding a `Main.run(String... argv)`?

Reply:
Also `"2"`